modules/microshift-proc-configuring-generic-device-plugin.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// microshift_configuring/microshift-gdp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-generic-device-plugin_{context}"]
+= Configuring the Generic Device Plugin
+
+:FeatureName: The Generic Device Plugin for {microshift-short}
+
+include::snippets/technology-preview.adoc[]
+
+The Generic Device Plugin (GDP) is disabled by default in {microshift-short}. To use the GDP, you must enable it and specify which host devices your Kubernetes applications can access. To enable the GDP, you must modify the {microshift-short} `config.yaml` file or create a configuration drop-in file. For example, `/etc/microshift/config.d/10-gdp.yaml`.
+
+.Prerequisites
+
+* You installed {microshift-short}.
+* You created a custom `config.yaml` file in the `/etc/microshift` directory.
+* You installed the {oc-first}.
+* You have `sudo` privileges on the {microshift-short} host.
+* You have identified the specific host devices that you want to expose to your {microshift-short} cluster. For example, `/dev/video0`, `/dev/ttyUSB*`, or USB Vendor/Product IDs.
+
+.Procedure
+
+. Open `/etc/microshift/config.yaml` in a text editor with `sudo` privileges.
```

Comment:
```suggestion
. Open `/etc/microshift/config.yaml` in a text editor with `sudo` privileges.
```
i think we mean the user needs sudo, not the editor, right? so something like, "From your CLI using `sudo` privileges, open a text editor:"